rush-plugins/rush-archive-project-plugin/.npmignore

```diff
@@ -8,29 +8,28 @@
 !/lib/**
 !/lib-*/**
 !/dist/**
+
+!CHANGELOG.md
+!CHANGELOG.json
+!heft-plugin.json
+!rush-plugin-manifest.json
 !ThirdPartyNotice.txt
-!/src/**
```

Comment:
@chengcyber Why is this repository including `/src/**` when publishing?

Reply:
`src` files are useful when sourcemap is released with the `dist` files. Developers can easily debug locally with the released version.